src/components/Feed.tsx

```diff
@@ -1,13 +1,56 @@
-import React from "react";
+import React, { useEffect } from "react";
+import * as sdk from "../services/sdk";
 import NewPost from "./NewPost";
 import PostList from "./PostList";
+import { useAppDispatch, useAppSelector } from "../redux/hooks";
+import * as server from "../services/server";
+import {
+  addNewFeedItems,
+  addNewFeedtoMainFeed,
+} from "../redux/slices/feedSlice";
+import { BaseFilters } from "@unfinishedlabs/sdk/dist/types/social/search";
+import { Button } from "antd";
 
 const Feed: React.FC = () => {
+  const [subscriptionID, setSubscriptionID] = React.useState<string>("");
+
+  const dispatch = useAppDispatch();
+  const newFeed = useAppSelector((state) => state.feed.newFeed);
+
+  useEffect(() => {
+    subscribe();
+    dispatch(addNewFeedtoMainFeed());
+    return unsubscribe;
+  }, []);
+
+  const subscribe = () => {
```

Comment:
if you have time perhaps move this into a hook

Reply:
I actually made the entire hook to do this and then realized there wasn't any need for a hook at all and the subscriptionID state could be removed entirely. It's only needed to be passed to the unsubscribe which we don't need state for in this case.